### DISCUSSION TITLE
AUR package

### DISCUSSION THREAD

**Discussion by ghost** [stumpapp/stump#97] [Ideas]
Hey, is there the possibility that we have in a future an AUR package?

**Comment by aaronleopold** [comment]
I'm all for it assuming it can be automated via CI. I use arch-based distros personally and there are already plans for CI to build binaries for linux (see https://github.com/aaronleopold/stump/issues/93). As for when it might happen, I'm not sure. Mostly because the linked CI issue is fairly dense as it is, but I am also just unfamiliar with the process for publishing to AUR. Regardless, I'll convert this discussion to an issue for better tracking 👍

**Reply by aaronleopold** [reply]
https://github.com/aaronleopold/stump/issues/98

**Reply by ghost** [reply]
Thanks! I think this is a promising project and I like that could be used with Tachiyomi. I don't have much experience in CI but I you need help about AUR I can give you a hand

**Reply by aaronleopold** [reply]
If it's something you're interested in trying to do, you're more than welcome to give it a go! There is definitely a lot to get done before a release is even considered so all help is appreciated

**Reply by dancamdev** [reply]
I was trying to get Stump running in a linux VM as well, mostly to spare myself from having to deal with docker volumes because moving files to them is tedious. While on a VM I'd ideally FTP into it and copy the files.

An AUR package would be awesome

**Reply by DieselTech** [reply]
Why not just mount a folder as a docker volume so you don't have to deal with moving things around?

**Comment by aaronleopold** [comment]
A community-maintained AUR package now exists: https://www.stumpapp.dev/docs/getting-started/installation/source#arch-linux-aur
